### ISSUE TITLE
Format numbers in statistics charts

### ISSUE THREAD

**Issue by saltcod** [supabase/supabase#21740]
The numbers on these charts are hard to parse.
We'd like to add formatting to make them more scannable

Eg: 9913 becomes 9,913
.toLocaleString should probably do it

![screenshot-2024-03-04-at-10 37 30](https://github.com/supabase/supabase/assets/105593/fed450ae-3aac-4b18-b493-09c94717e7b3)

**Comment by thefool76** [comment]
hey there! I would like to work on it. can you please assign it ?

**Comment by uchechiukpa** [comment]
If this has not been assigned yet, I would like to work on it.

**Comment by saltcod** [comment]
Got a PR ^^ reviewing rn! thanks all
